### PR TITLE
Fix #8 Added Recycler View item layout

### LINKED ISSUE #8: XML Layout for Recycler View Item in User and Admin Dashboard
Create the XML layout for User Dashboard and Admin Dashboard based on the design provided in the [Figma](https://www.figma.com/design/jYm7XGidk1L3HNf9e4Otwn/myLeave?node-id=6-18&t=HjrOtyAfoLbxQDPn-1). 

Your layout must follow these guidelines:

1. Use Linear Layout to create the Recycler View item, no need to create the adapter .
4. Adhere to the given Figma design
5. Follow the specified naming convention for each view group ID to maintain consistency and clarity.

### PR DESCRIPTION
**Pull request by ayushsaksena30**
## Overview

1. This PR fixes or fixes part of #8.
2. This PR does the following: Uses Nested Linear Layout and Card View to implement UI styles from the [Figma](https://www.figma.com/design/jYm7XGidk1L3HNf9e4Otwn/myLeave?node-id=6-18&t=Ubw9NPNwzQCMmjhl-0) for User and Admin Dashboard.

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
Screenshot 2024-06-09 at 11 49 57 PM

## PR Pointers

### CODE REVIEW

File: app/src/main/res/layout/item_applications.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            android:padding="16dp"
+            android:orientation="vertical">
+
+            <TextView
+                android:id="@+id/tv_leaveType"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textColor="@color/grey"
+                android:text="@string/leave_type"
+                android:textSize="16sp" />
```
Comment (Agrim2112) [inline app/src/main/res/layout/item_applications.xml:27]: Add the font for all the Text Views.

File: app/src/main/res/layout/item_applications.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        android:layout_height="wrap_content"
+        app:cardCornerRadius="12dp"
+        android:layout_margin="12dp"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="16dp"
+            android:orientation="vertical">
```
Comment (Agrim2112) [inline app/src/main/res/layout/item_applications.xml:19]: Make the orientation horizontal. Add 2 nested Linear layouts within this layout, one will contain tvLeaveType and tvLeaveDuration and other will contain the application status.

**Review by Agrim2112 — CHANGES_REQUESTED** [review]
Make the requested changes

**Comment by ayushsaksena30** [comment]
@Agrim2112 I have made the necessary changes and pushed the new commit into the PR.

**Comment by ayushsaksena30** [comment]
@Agrim2112 I have made the necessary changes and pushed the new commit into the PR.

**Pull request merged**